TsRandomizer/Screens/Settings/GameSetting.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace TsRandomizer.Screens.Settings
+{
+	public abstract class GameSetting
+	{
+
+		[JsonProperty]
+		public dynamic CurrentValue { get; protected set; }
```

Comment:
dynamic is not the not the right type for a type you dont know what it is, for such types we use object as that is the base of all type